connector/connect/common/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -774,11 +774,11 @@ message RepartitionByExpression {
   optional int32 num_partitions = 3;
 }
 
-message FrameMap {
-  // (Required) Input relation for a Frame Map API: mapInPandas, mapInArrow.
+message MapPartitions {
```

Comment:
How about MapBatches instead, @hvanhovell?

Reply:
MapPartitions works to me too if you prefer this one more.

Reply:
I prefer MapPartitions because that reflects what is actually does. Batch is IMO a property of the UDF used.